### ISSUE TITLE
More line space from crengine

### ISSUE THREAD

**Issue by houqp** [koreader/kindlepdfviewer#284]
Current Crengine only support 200% line space. However, one user reports to me that for one of his book, 200% is still not large enough. So it will be better if we can make the line space more adjustable.

This is somehow a upstream issue.

**Comment by kkzxak47** [comment]
Thanks houqp. I am the "one user"...and here I am.

And yet I got another problem with the latest release: partial refresh. 
By default it will now go through 5 pages before a full refresh is done. But I'm too picky for this. I got to do full refresh on every single page.

As a non-programmer, I have to check wiki ("partial_refresh_count" won't work anymore) and search for this problem in "Issues", figure out what exactly are "rcount" and "rcountmax"...

So, in a word, I wonder if it is possible for an average user to change these settings in a more intuitive way. Maybe you can add a hotkey for "pages before refreshing"? Just like the ones you use to change font size and line spacing.

**Comment by tigran123** [comment]
@kkzxak47 Yes, you just press Shift-R and set the value to 0. You don't need to edit any files manually.

Btw, in kpdfdjview the value is 0 and is not changeable (because I believe that should be the normal behaviour of ebook readers --- no ghosts, please!)

**Comment by kkzxak47** [comment]
@tigran123 Got it. Thanks for the tip. I should have checked the help page more carefully...

Thank you for your work! This project is great!

**Comment by dpavlin** [comment]
I added it to it to changelog at download page, so I'm closing this issue.

**Comment by houqp** [comment]
@dpavlin , are you closing this because of the `Shift-R` feature?

**Comment by dpavlin** [comment]
@houqp yes :-) I got carried away by last @kkzxak47 comment...

Changing minimum and maximum line spacing is easy, list modify `self.line_space_percent = math.max(self.line_space_percent, something)` lines in `crereader.lua`.

What would sane minimum and maximum be? (currently they are 80 - 200)
